DemoApp/StreamChat/Components/CustomAttachments/DemoChatMessageListVC.swift

```diff
@@ -0,0 +1,7 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatUI
+
+class DemoChatMessageListVC: ChatMessageListVC {}
```

Comment:
Right now, we don't have any other customization here, but we do have an implementation in `LocationAttachmentViewDelegate`. The extension is there, so that all location attachment code is inside the Location folder